doc/contributor/threads.md

```diff
@@ -1,18 +1,18 @@
 # Threads
 
-Each Ruby fiber is implemented as a Java thread. A Ruby thread is implemented as
-a group of one or more fibers. A Ruby thread has a root fiber which is created
-even if you aren't explicitly using fibers, so you are always running in a
-fiber.
+Each Ruby Fiber is implemented as a Java thread.
+In Ruby, a Thread has one or more Fibers.
```

Comment:
I think the mixed casing of `Thread` & `thread` and `Fiber` and `fiber` is a bit confusing. If you're talking about the class, capitalizing the word makes sense (any maybe wrap in backticks). If just talking about the general concept, lower-case makes sense. I'm having a hard time working out the convention here.

Reply:
You are right, fixed to use lower case everywhere as it's about the concepts.